recipe/meta.yaml

```diff
@@ -24,26 +24,17 @@ requirements:
   host:
     - python
     - pip
-    - urllib3
-    # secure
-    - pyopenssl >=0.14
```

Comment:
on CF all those deps are still there, are they really not needed ?

Reply:
I don't see anything else listed in the upstream repo

https://github.com/jelmer/dulwich/blob/2c9cf9de8266aee2b966fb1f89eb6c55fbcc7a23/pyproject.toml#L2

https://github.com/jelmer/dulwich/blob/master/requirements.txt

Reply:
see this file: https://github.com/conda-forge/dulwich-feedstock/blob/main/recipe/meta.yaml

Reply:
Those dependencies are from 2018 https://github.com/conda-forge/dulwich-feedstock/commit/b4afda2e5c3827d2eaed4631524e9ba5fcefd5cf and now they are not required